lib/book.py

```diff
@@ -443,28 +281,27 @@ def create_xray(self, aConnection, sConnection, info, log=None, abort=None, remo
             return
         if log: log('%s \t\tCreating x-ray...' % datetime.now().strftime('%m-%d-%Y %H:%M:%S'))
         self._write_xray(info, remove_files_from_dir=remove_files_from_dir)
-        return (aConnection, sConnection)
+        return connection
```

Comment:
I've never found this connection handling very clear, but maybe there is a reason for it.  It seems odd/unpythonic to pass a variable into a function and then pass out what is possibly the same.  Should probably be set on some object that all these functions can see.

Not a big deal, and not changed by this, just pointing out something a bit odd..